twitter4j-core/src/main/java/twitter4j/TwitterImpl.java

```diff
@@ -257,6 +264,114 @@ public UploadedMedia uploadMedia(String fileName, InputStream image) throws Twit
                 , new HttpParameter("media", fileName, image)).asJSONObject());
     }
 
+
+    @Override
+    public UploadedMedia uploadMediaChunked(File mediaFile) throws TwitterException {
+        checkFileValidity(mediaFile);
+        try {
+            return uploadMediaChunked(mediaFile.getName(), new FileInputStream(mediaFile), mediaFile.length());
```

Comment:
InputStream is never closed.

Reply:
Yikes, that's embarrassing :worried:. That actually does not look like something I'd write, but here we are (or maybe I inherited this code, as this PR is basically a remix of existing code). Whatever, thanks for spotting this :thumbsup:

Reply:
Yikes, that's embarrassing :worried:. That actually does not look like something I'd write, but here we are (or maybe I inherited this code, as this PR is basically a remix of existing code). Whatever, thanks for spotting this :+1: